mimic/model/clb_objects.py

```diff
@@ -490,6 +496,10 @@ def delete_nodes(self, lb_id, node_ids):
         if lb_id not in self.lbs:
             return not_found_response("loadbalancer"), 404
 
+        if len(node_ids) > self.batch_delete_limit:
```

Comment:
You are absolutely right. It validates the IDs before checking for limit. I'll do the same thing here.